### ISSUE TITLE
[microshift] `crc oc-env` should take care of `export KUBECONFIG=...`

### ISSUE THREAD

**Issue by cfergeau** [crc-org/crc#3591]
### General information

  * OS: Linux

After running `crc start` with the `microshift` preset:
```
INFO Waiting for kube-apiserver availability... [takes around 2min] 
Started the MicroShift cluster.

export KUBECONFIG="/home/teuf/.crc/machines/crc/kubeconfig"

Use the 'oc' command line interface:
  $ eval $(crc oc-env)
  $ oc COMMAND
```

```
$ crc oc-env
export PATH="/home/teuf/.crc/bin/oc:$PATH"
# Run this command to configure your shell:
# eval $(crc oc-env)
```

`crc oc-env` could also handle the `export KUBECONFIG=...` instead of expecting the user to do it.

**Comment by praveenkumar** [comment]
Instead of making it export as part of `oc-env`, we should append this kubeconfig file to the user $KUBECONFIG env or `~/.kube/config` and then make microshift as default context. With doing that all other tool able to use same context instead exporting and also in the podman desktop side it will able to see it in the context window.

**Comment by anjannath** [comment]
fixed in #3597
